guide/getting-started/software.md

```diff
@@ -36,66 +36,74 @@ Illustration sources
+The unique history and architecture of Bitcoin has resulted in several categories of software, with each one serving a distinct purpose. Provided below is an overview of standard and specialized application categories.
 
-## Command-line interfaces (CLI)
-
-Primarily used by developers, Bitcoin CLIs offer feature-rich ways to interact with Bitcoin. They can be thought of as general-purpose toolboxes, so they require certain technical knowledge and are not optimized for specific use cases. New features are often first included in a CLI and then later added to graphical user interfaces (GUI).
-
-{% include image.html
-   image = "/assets/images/guide/getting-started/software/cli-example.jpg"
-   retina = "/assets/images/guide/getting-started/software/cli-example@2x.jpg"
-   alt-text = "Illustrative interface of a command-line interaction"
-   caption = 'Command line interfaces are text-only. Texture by [Joel Filipe](https://unsplash.com/@joelfilip){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
-   width = 800
-   height = 350
-%}
-
-## Block explorers
+## Wallets
 
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/getting-started/software/explorer-example.jpg"
-   retina = "/assets/images/guide/getting-started/software/explorer-example@2x.jpg"
-   alt-text = "Illustrative interface for block explorer software"
-   caption = 'Block explorer offer insight into activiy on the bitcoin network. Texture by [Mike van den Bos](https://unsplash.com/@mike_van_den_bos){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
+   image = "/assets/images/guide/getting-started/software/wallet-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/wallet-example@2x.jpg"
+   alt-text = "Illustrative interface of a wallet application"
+   caption = "Wallets allow users to send, receive, and manage transactions."
    width = 400
-   height = 388
+   height = 396
    layout = "float-right-desktop"
 %}
 
-If blockchain is a public database, block explorers are simply windows into that database. They let users view data about specific transactions, latest blocks, block height, and so on.
+Wallets are perhaps the most important Bitcoin applications. They provide easy-to-use graphical user interfaces (GUIs) designed for broad audiences that let bitcoin owners [send]({{ '/guide/payments/send/' | relative_url }}), [receive]({{ '/guide/payments/receive/' | relative_url }}) store and manage their coins.
+
+Wallet features vary by application but always include wallet setup, balance and transaction records, and the ability to send and receive bitcoin. The full range of features that wallets may support is broad and includes security and privacy options, currency exchange features, accounting tools, interoperability, accessibility, and localization options.
```

Comment:
```suggestion
Wallet features vary by application but always include [wallet setup]({{ '/guide/onboarding/introduction/' | relative_url }}), balance and transaction records, and the ability to [send]({{ '/guide/payments/send/' | relative_url }}) and [receive]({{ '/guide/payments/receive/' | relative_url }}) bitcoin. The full range of features that wallets may support is broad and includes security and [privacy]({{ '/guide/payments/privacy/' | relative_url }}) options, currency exchange features, accounting tools, [interoperability]({{ '/guide/designing-products/principles/#interoperability' | relative_url }}), accessibility, and localization options.
```
What do you think about adding all these cross-references? Since this is an introductory page, it might be appropriate to have a lot of links, in case something sparks the readers interest.

Reply:
We could also links to the "Common user flows" page, which remains a bit more top-level than linking directly to the Payments section, etc.

Reply:
Links are good, changed the onboarding to the Common user flow page, better start.